### ISSUE TITLE
Can you add a path options field?

### ISSUE THREAD

**Issue by jacobtipp** [nelsonic/hapi-register#28]
I really like the simplicity of using a register/login plugin with hapi.js.

The only thing is my application may need a custom path other than `/register`.

I feel like this should be an optional field.

**Comment by nelsonic** [comment]
@traducer yes, it should really be an optional parameter. e.g: `options.registerPath` ?

**Comment by jacobtipp** [comment]
@nelsonic, yeah that would work just fine, since hapi-login is almost identical, perhaps it should also have an optional path? I can make a pull request if you like.

**Comment by nelsonic** [comment]
@traducer if you have time, please do create PR. 
ideally we want:

``` js
server.route({
      method: '*',
      path: options.registerPath || '/register', // default to /register
      config: {
        auth: false,
        validate: {
          payload : options.fields,
          failAction: options.fail_action_handler || 'error'
        }
      },
      handler: options.handler
  });
```

We need tests for both though...

**Comment by jacobtipp** [comment]
@nelson, Yeah writing the tests with tape should be no problem, I glanced at your other tests so I'll try to keep mine similar to them.

**Comment by adam-beck** [comment]
This can be closed, right?

**Comment by nelsonic** [comment]
@adam-beck / @traducer, yes this was resolved by #29 👍
